package.json

```diff
@@ -144,7 +144,8 @@
     "trim-newlines": "^3.0.1",
     "glob-parent": ">=5.1.2",
     "nth-check": ">=2.0.1",
-    "trim": ">=0.0.3"
+    "trim": ">=0.0.3",
+    "terser": "^5.14.2"
```

Comment:
nit, it would be good idea to add the specific dependency of ours that is depending on `terser` as wrapper, so we know when to get rid of it easier.

Reply:
I will try to do that